tools/install-tcmalloc.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+#
+# Script to install libtcmalloc.so in the current WiredTiger git
+# repository.
+#
+# This is serves the same purpose, but is NOT THE SAME as:
+#
+# test/evergreen/tcmalloc_install_or_build.sh
+#
+# and the tcmalloc source version referenced here should be updated in
+# sync with that script.
+
+set -euf -o pipefail
+
+die() { echo "$@"; exit 1; }
+
+HERE=$(git rev-parse --show-toplevel) || die "FATAL Run in root of git workspace"
+[[ -f ${HERE}/CMakeLists.txt ]] || die "Does not look like a WT workspace - no CMakeLists.txt"
+[[ -d ${HERE}/TCMALLOC_LIB ]] && die "TCMALLOC_LIB already present"
```

Comment:
Maybe it's better to check for the presence of `TCMALLOC_LIB/libtcmalloc.so`, not just directory. Build could fail or be aborted.

Reply:
I'd rather that the user be forced to verify and delete the directory, rather than try to be too precise, as this script generates other files.
If we are running this script *a lot* something has gone terrribly wrong.